actors/builtin/miner/miner_actor.go

```diff
@@ -605,6 +614,18 @@ func (a Actor) checkTemporaryFaultEvents(rt Runtime, sectors *abi.BitField) {
 	sectorNos := bitfieldToSectorNos(rt, sectors)
 
 	rt.State().Transaction(&st, func() interface{} {
+
+		maxAllowedFaults, err := st.GetMaxAllowedFaults(store)
+		if err != nil {
+			rt.Abortf(exitcode.SysErrInternal, "failed to get number of sectors")
+		}
+		faultsMap, err := st.FaultSet.AllMap(maxAllowedFaults)
+		if err != nil {
+			rt.Abortf(exitcode.ErrIllegalState, "too many faults")
+		}
+
+		unsets := make(map[uint64]struct{})
```

Comment:
This looks to be unused.

Reply:
Yeah, it is. Remainder of earlier version.